Assets/MRTK/Providers/XRSDK/XRSDKBoundarySystem.cs

```diff
@@ -26,6 +26,17 @@ public class XRSDKBoundarySystem : BaseBoundarySystem
         {
         }
 
+        /// <inheritdoc/>
+        protected override bool IsXRDevicePresent
+        {
+            get
+            {
+                List<InputDevice> devices = new List<InputDevice>();
+                InputDevices.GetDevicesWithCharacteristics(InputDeviceCharacteristics.HeadMounted, devices);
+                return devices.Count <= 0;
```

Comment:
I think this needs to be flipped now, since a device being present means that this count should be greater than 0:

```suggestion
                return devices.Count > 0;
```

Reply:
Thanks for the catch!